lua/gt.lua

```diff
@@ -16,8 +16,13 @@ return function (net_conf, numa_table)
 		n_lcores)
 	gatekeeper.gt_assign_lcores(gt_conf, gt_lcores)
 
+	local net_prefix = "10.0.2.1/24"
+	local net_prefix6 = "2003:db8::1/32"
```

Comment:
What do these network prefixes represent? How are they different from what is assigned in `lua/net.lua`?